### ISSUE TITLE
Figure out file structure to accommodate apps code

### ISSUE THREAD

**Issue by georgeblahblah** [guardian/dotcom-rendering#7057]
Where should the code for rendering for apps live inside DCR?

**Comment by georgeblahblah** [comment]
Started an [RFC](https://github.com/guardian/dotcom-rendering/issues/7174) to collect opinions.

To unblock progress now, we will take a similar approach used in the [spike](https://github.com/guardian/dotcom-rendering/pull/6818) and create an `apps/` subdirectory of `src/`.

Later, we can refactor and test whether the proposed solutions helps with any pain points.

**Comment by rhiannareechaye** [comment]
This is a 'nice to have' for our milestone but it's not a blocker for other teams contributing to the platform

**Comment by OllysCoding** [comment]
Steps I plan to take in order to create a new structure:

- [ ] Update the 'server' structure in DCR so that it's better orientated for the apps solution
- [ ] Rename 'browser' to 'client' in DCR
- [ ] Move the apps specific code we've developed back into web directory with `.web` and `.apps` in filenames
- [ ] Consider next steps:
     - Use `.amp` filenames and move amp code into main web directory?
     - Lift web to top level? (do we leave amp or bring it to the top level as well?)

**Comment by rhiannareechaye** [comment]
Hey @OllysCoding, can we close this ticket?

**Comment by VDuczekW** [comment]
we are closing as work has been done
